scripts/changesets/generate-releases.js

```diff
@@ -54,11 +54,20 @@ const getReleasedPackages = async (csOutput, pkgs) => {
   }, []);
 };
 
+function setNpmToken() {
+  const env = process.env;
+  const registry = '//registry.npmjs.org/';
+  childProcess.execSync(
+    `npm config set ${registry}:_authToken=${env.NPM_TOKEN}`,
```

Comment:
I think that might expose the token to logs, I would do it in circleci config so they can hide it.
https://circleci.com/docs/deploy-to-npm-registry